docs/api/apiv3/components/schemas/watchers_model.yml

```diff
@@ -1,6 +1,36 @@
 # Schema: WatchersModel
 ---
-type: object
+allOf:
+  - "$ref": "./collection_model.yml"
+  - type: object
+    required:
+      - _links
+      - _embedded
+    properties:
+      _links:
+        type: object
+        required:
+        - self
+        properties:
+          self:
+            allOf:
+            - "$ref": "./link.yml"
+            - description: |-
+                The watcher list
+                
+                **Resource**: WatchersModel
+              readOnly: true
+      _embedded:
+        type: object
+        properties:
+          elements:
+            type: array
+            readOnly: true
+            items:
+                allOf:
+                - "$ref": "./user_model.yml"
+                - description: Collection of Users
+                readOnly: true
```

Comment:
`readOnly` is misplaced here. From OpenAPI spec for v3.0.3: `Relevant only for Schema "properties" definitions.`

I'd say, remove it from here. Only thing you need is `readOnly` in line 28, which already states, that the whole array `elements` should only be validated for response bodies, and no request bodies.

Reply:
Adjusted.